modules/iam/main.tf

```diff
@@ -9,8 +14,7 @@ resource "aws_iam_role" "sagemaker_role" {
       "Effect": "Allow",
       "Principal": {
         "Service": [
-          "sagemaker.amazonaws.com",
-          "lambda.amazonaws.com"
```

Comment:
There are no lambdas created by this module. If a user wants a lambda to be able to assume this role, they should have code for that outside of this module.